docs/main/user-guides/upgrades.md

```diff
@@ -73,6 +73,10 @@ This section describes the Replicated Storage upgrade from OpenEBS Umbrella char
 
 1. Start the helm upgrade process with the new chart, i.e. 4.0.0 by using the below command:
 
+:::caution
+For users who needs hot upgrade, `--set mayastor.agents.core.rebuild.partial.enabled=false` is mandatory while upgrading to OpenEBS 4.0.0 to ensure there are no issues in the upgrade process. For users who can scale down the applications in the upgrade process can skip using this flag. 
```

Comment:
It is highly recommended to disable partial rebuild during upgrade to OpenEBS 4.0.0 to ensure data consistency during upgrade. Pass the value `..` in the upgrade command.